### ISSUE TITLE
donejs deploy doesn't run on windows

### ISSUE THREAD

**Issue by matthewp** [donejs/donejs#487]
```
PS C:\Users\Matthew\dev\place-my-order> donejs deploy
The system cannot find the path specified.
Error: Command `C:\Users\Matthew\dev\place-my-order\node_modules\.bin\deploy` did
not complete successfully
    at ChildProcess.<anonymous> (C:\Users\Matthew\dev\place-my-order\node_modules\
donejs-cli\lib\utils.js:37:23)
    at emitTwo (events.js:87:13)
    at ChildProcess.emit (events.js:172:7)
    at ChildProcess.cp.emit (C:\Users\Matthew\dev\place-my-order\node_modules\cros
s-spawn-async\lib\enoent.js:37:29)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:200:12)
PS C:\Users\Matthew\dev\place-my-order>
```

Directly calling `node_modules\.bin\deploy` it does work, so must be something with the way it's being called.

**Comment by matthewp** [comment]
This is the cause: https://github.com/donejs/cli/issues/32

**Comment by matthewp** [comment]
Fixed in donejs-cli 0.6.5
